content/applicationHost.xdt

```diff
@@ -54,4 +54,11 @@
       </environmentVariables>
     </runtime>
   </system.webServer>
+  <location path="%XDT_SITENAME%" xdt:Locator="Match(path)">
+    <system.webServer xdt:Transform="InsertIfMissing">
```

Comment:
Just curious: why didn't this go into the `<system.webServer>` section above?

Reply:
I **think** (since I'm no expert in XML Transforms) that this makes sure that it only gets added to the main site instead of every single "site" that could be running on the Azure App Service

Reply:
I also followed the Kudu documentation for this: https://github.com/projectkudu/kudu/wiki/Xdt-transform-samples#registering-an-iis-managed-httpmodule

Reply:
Thanks!